src/containers/shared/components/Transaction/types.ts

```diff
@@ -1,44 +1,23 @@
 import React from 'react'
 import { Memo } from '../../transactionUtils'
 
-export interface Instructions {
```

Comment:
Why was this type removed?

Reply:
It was left over from the old transactions scheme.  It was never referenced.